kubeconfig-creator/example/pipeline.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: kubeconfig-test-pipeline
+spec:
+  workspaces:
+    - name: shared-workspace
+  tasks:
+    - name: kubeconfig-creator
+      taskRef:
+        name: kubeconfig-creator
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+      params:
+        - name: name
+          value: cluster-bot
+        - name: username
+          value: kubeadmin
+        - name: url
+          value: https://api.ci-ln-vsqykgk-d5d6b.origin-ci-int-aws.dev.rhcloud.com:6443
```

Comment:
Just checking here: do you intend to expose this address?  Is it for a real cluster?

And are the cadata / clientKeyData / clientCertificateData real as well?  It seems like this might have leaked credentials?

Reply:
Actually, it was cluster-bot credentials. I intend to expose these, just to show example to the user. Will add demo data here.